rustls-platform-verifier/src/verification/others.rs

```diff
@@ -62,18 +68,23 @@ impl Verifier {
     }
 
     // Attempt to load CA root certificates present on system, fallback to WebPKI roots if error
-    fn init_verifier(&self) -> Result<WebPkiVerifier, TlsError> {
+    fn init_verifier(&self) -> Result<Arc<WebPkiServerVerifier>, TlsError> {
         let mut root_store = rustls::RootCertStore::empty();
 
         // For testing only: load fake root cert, instead of native/WebPKI roots
         #[cfg(any(test, feature = "ffi-testing", feature = "dbg"))]
         {
             if let Some(test_root) = &self.test_only_root_ca_override {
-                let (added, ignored) = root_store.add_parsable_certificates(&[test_root.clone()]);
+                let (added, ignored) =
+                    root_store.add_parsable_certificates([pki_types::CertificateDer::from(
+                        test_root.clone(),
```

Comment:
Since the `test_roots` are `&'static`, this clone may not be needed?

Reply:
Fixed.